osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -51,6 +51,16 @@ public class BeatmapInfo : IEquatable<BeatmapInfo>, IJsonSerializable, IHasPrima
         [NotMapped]
         public BeatmapOnlineInfo OnlineInfo { get; set; }
 
+        /// <summary>
+        /// The playable length of this beatmap.
```

Comment:
need to say milliseconds or seconds as it isn't obvious (ditto for `MaxLength` below)

Reply:
case in point:

![image](https://user-images.githubusercontent.com/191335/60896230-ef041500-a2a0-11e9-8a7c-624fc952a19b.png)

![image](https://user-images.githubusercontent.com/191335/60896287-0cd17a00-a2a1-11e9-950f-cbfcc59e301d.png)

Reply:
why did you use `TimeSpan.FromSeconds` in `SongBar.cs`, won't that give you a time span based on the number of seconds you passed?

Reply:
I'm saying one of the above two is incorrect.